### DISCUSSION TITLE
reset password

### DISCUSSION THREAD

**Discussion by Eddieua** [aaronwmorris/indi-allsky#1194] [General]
I am going to config the camera with web-based interface. However, I have forgotted the indi-allsky username / password.
How can I reset it without reinstalling everything?

thank you,

**Comment by aaronwmorris** [comment]
You can reset the password with the following instructions:

https://github.com/aaronwmorris/indi-allsky/wiki/Web-Interface-Password

**Reply by Eddieua** [reply]
it works!  Thank you aaronwmorris.
